Views/Account/Login.cshtml

```diff
@@ -1,73 +1,90 @@
 @model Login
 
-<style>
-    body {
-        background-image: url('https://upload.wikimedia.org/wikipedia/commons/b/ba/New_Punjab_Club_food_spread.jpg');
-        background-size: cover;
-        background-position: center;
-        background-attachment: fixed;
-        margin: 0;
-        height: 100vh;
-        background-color: gray;
-    }
-    .container {
-        /* Dark overlay for better text visibility */
-        padding: 2rem;
-        border-radius: 8px;
-        max-width: 450px;
-        /* Adjust width as needed */
-        margin: 0 auto;
-        /* Center horizontally */
-        margin-top: 5rem;
-        color: white;
-        /* Adds space from top */
-    }
-    .form-group {
-        margin-bottom: 1rem;
-    }
-    label {
-        font-weight: bold;
-    }
-    .btn-primary {
-        margin-top: 1rem;
-        margin-left: 110px;
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
remove these tags and script tag at the bottom because its already added in main file